quaistats/quaistats.go

```diff
@@ -457,7 +475,7 @@ func (s *Service) handleBlock(block *types.Block) {
 		s.appendTimeStatsQueue.Enqueue(appStats)
 	}
 
-	if block.NumberU64()%c_txBatchSize == 0 && s.sendfullstats {
+	if block.NumberU64()%c_txBatchSize == 0 && s.sendfullstats && len(block.Header().Location()) == 2 {
```

Comment:
please use the location method: `func (loc Location) Context()` to check the context of location

Reply:
Fixed